tools.sh

```diff
@@ -66,3 +66,6 @@ fi
 if ! rpm -q libguestfs-xfs; then
     sudo yum install libguestfs-xfs
 fi
+
+# Restart the libvirt service after update
+sudo systemctl restart libvirtd
```

Comment:
Could you run a `yum update`+`systemctl restart libvirtd` in the GCP job before invoking `snc.sh`? Or even update the CI image used? I don't think it's snc.sh job to fix outdated CI images.